niworkflows/anat/ants.py

```diff
@@ -0,0 +1,555 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""
+Nipype translation of ANTs workflows
+------------------------------------
+
+"""
+from __future__ import print_function, division, absolute_import, unicode_literals
+
+import os
+from multiprocessing import cpu_count
+from pkg_resources import resource_filename as pkgr_fn
+from packaging.version import parse as parseversion, Version
+from pathlib import Path
+from ..data import TEMPLATE_MAP, get_dataset
+from ..nipype.pipeline import engine as pe
+from ..nipype.interfaces import utility as niu
+from ..nipype.interfaces.fsl.maths import ApplyMask
+from ..nipype.interfaces.ants import N4BiasFieldCorrection, Atropos, MultiplyImages
+from ..interfaces.ants import (
+    ImageMath,
+    ResampleImageBySpacing,
+    AI,
+    ThresholdImage,
+)
+from ..interfaces.fixes import (
+    FixHeaderRegistration as Registration,
+    FixHeaderApplyTransforms as ApplyTransforms,
+)
+
+ATROPOS_MODELS = {
+    'T1': (3, 1, 2, 3),
+    'T2': (3, 3, 2, 1),
+    'FLAIR': (3, 1, 3, 2),
+}
+
+
+def brain_extraction(name='antsBrainExtraction',
```

Comment:
Should we stick with our convention `init_brain_extraction_wf`?

Reply:
sounds good